Yuliia/most-probable-letters.js

```diff
@@ -0,0 +1,103 @@
+const fs = require('fs');
+const readline = require('readline');
+
+if (process.argv.length < 4) {
+  console.error('Example: node statistics-counter.js filename.txt letter');
```

Comment:
In this handler we check number of args and if values les then 4, we raise error and exit?

Reply:
Yes. There are supposed to be 4 args to start the program.  This is a standard check